Description: fix FTFBS with glibc2.32 due to rpc vs tirpc
 Found by the full groovy rebuild this FTBFS is due to
 glibc2.32 dropping the option to enable the outdated
 compat rpc support.
 That means finally everything has to use tirpc for real.
 In the case of open-vm-tools a few makefiles needed to
 get the right variables (already detected by pkg-config)
 propagated.
Forwarded: yes (https://github.com/vmware/open-vm-tools/pull/469)
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Last-Update: 2020-09-29
--- a/open-vm-tools/tests/testDebug/Makefile.am
+++ b/open-vm-tools/tests/testDebug/Makefile.am
@@ -22,6 +22,7 @@ libtestDebug_la_CPPFLAGS =
 libtestDebug_la_CPPFLAGS += @CUNIT_CPPFLAGS@
 libtestDebug_la_CPPFLAGS += @GOBJECT_CPPFLAGS@
 libtestDebug_la_CPPFLAGS += @PLUGIN_CPPFLAGS@
+libtestDebug_la_CPPFLAGS += @XDR_CPPFLAGS@
 
 libtestDebug_la_LDFLAGS =
 libtestDebug_la_LDFLAGS += @PLUGIN_LDFLAGS@
@@ -30,6 +31,7 @@ libtestDebug_la_LIBADD =
 libtestDebug_la_LIBADD += @CUNIT_LIBS@
 libtestDebug_la_LIBADD += @GOBJECT_LIBS@
 libtestDebug_la_LIBADD += @VMTOOLS_LIBS@
+libtestDebug_la_LIBADD += @XDR_LIBS@
 libtestDebug_la_LIBADD += ../vmrpcdbg/libvmrpcdbg.la
 
 libtestDebug_la_SOURCES =
--- a/open-vm-tools/tests/testPlugin/Makefile.am
+++ b/open-vm-tools/tests/testPlugin/Makefile.am
@@ -22,6 +22,7 @@ libtestPlugin_la_CPPFLAGS =
 libtestPlugin_la_CPPFLAGS += @CUNIT_CPPFLAGS@
 libtestPlugin_la_CPPFLAGS += @GOBJECT_CPPFLAGS@
 libtestPlugin_la_CPPFLAGS += @PLUGIN_CPPFLAGS@
+libtestPlugin_la_CPPFLAGS += @XDR_CPPFLAGS@
 
 libtestPlugin_la_LDFLAGS =
 libtestPlugin_la_LDFLAGS += @PLUGIN_LDFLAGS@
--- a/open-vm-tools/tests/vmrpcdbg/Makefile.am
+++ b/open-vm-tools/tests/vmrpcdbg/Makefile.am
@@ -21,6 +21,7 @@ libvmrpcdbg_la_CPPFLAGS =
 libvmrpcdbg_la_CPPFLAGS += @CUNIT_CPPFLAGS@
 libvmrpcdbg_la_CPPFLAGS += @GMODULE_CPPFLAGS@
 libvmrpcdbg_la_CPPFLAGS += @VMTOOLS_CPPFLAGS@
+libvmrpcdbg_la_CPPFLAGS += @XDR_CPPFLAGS@
 libvmrpcdbg_la_CPPFLAGS += -I$(top_srcdir)/lib/rpcChannel
 
 libvmrpcdbg_la_LDFLAGS =
